src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/ItemMapperTest.java

```diff
@@ -31,7 +30,7 @@
     private static final String PREFIX = "PREFIX-";
     private static final ItemMapper SUT = new ItemMapper(new TableMapping(new DynamoTableDescriptionImpl(
         CreateTableRequestBuilder.builder().withTableKeySchema("virtualhk", S).build()),
-        virtualTableDescription1 -> Optional.of(new DynamoTableDescriptionImpl(CreateTableRequestBuilder.builder()
+            new SingletonCreateTableRequestFactory(new DynamoTableDescriptionImpl(CreateTableRequestBuilder.builder()
```

Comment:
Had to introduce this class, since CreateTableRequestFactory is no longer a functional interface (see comments above).